penny_university/settings/base.py

```diff
@@ -60,7 +60,8 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'penny_university.middleware.DebugPassthrough',
+    'penny_university.middleware.debug_passthrough.DebugPassthrough',
+    'penny_university.middleware.integration_test_logging.IntegrationTestLogging',
```

Comment:
I think our logging is currently to standard out. There may be situations where we want to log to a file to commit for the frontend to use in tests.

Reply:
True and true. Feel free to upgrade us when you have need. The logging at this point is very naive.